src/helpers/manage-merge-queue.ts

```diff
@@ -116,7 +116,7 @@ const getQueuedPullRequests = async (): Promise<PullRequestList> => {
 
 export const enableAutoMerge = async (pullRequestId: string, mergeMethod = 'SQUASH') => {
   try {
-    return await octokitGraphql(`
+    await octokitGraphql(`
```

Comment:
Why are we removing `return` here?

Reply:
Just wanted to remove cases of `return await` since it's kind of confusing to me 😂

Reply:
If we are not using the response returned, then yeah getting rid of `return` makes sense.